src/main/java/com/agency04/battleship/mapper/PlayerDTOMapper.java

```diff
@@ -0,0 +1,34 @@
+package com.agency04.battleship.mapper;
+
+import com.agency04.battleship.dto.PlayerDTO;
+import com.agency04.battleship.model.Player;
+import com.agency04.battleship.repository.PlayerRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PlayerDTOMapper {
```

Comment:
nazovi smamo PlayerMapper jer ovako implicira da je to isključivo u DTO mapper (a postojati će metode koje radi i obrnuto)